thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -615,9 +627,16 @@ public Response tuneAlertFilter(@PathParam("functionId") long id,
     }
 
     // write to DB
-    autotuneConfig.setFunctionId(id);
-    autotuneId = DAO_REGISTRY.getAutotuneConfigDAO().save(autotuneConfig).toString();
-    return Response.ok(autotuneId).build();
+    List<Long> autotuneIds = new ArrayList<>();
+    for (long functionId : anomalyFunctionIds) {
+      autotuneConfig.setId(null);
```

Comment:
why setting Id to null?

Reply:
It is for create a new AutotuneConfig entry for each anomaly function.